src/components/vuestic-components/va-radio/VaRadio.vue

```diff
@@ -108,135 +138,141 @@ export default {
     },
   },
   methods: {
-    onClick () {
+    click (e) {
       if (!this.disabled) {
-        this.$emit('input', this.option)
+        return this.$emit('input', this.option, e)
       }
     },
-    onFocus (e) {
+    focus (e) {
       if (!this.disabled) {
-        this.$emit('focus', e)
+        return this.$emit('focus', e)
       }
     },
+    validate () {
+      return true
+    },
+    clear (e) {
+      return this.$emit('input', null, e)
```

Comment:
I don't completely understand why would you need to pass native event.

Reply:
Aaand I still don't. Please explain :).

Reply:
fixed